dask/bytes/core.py

```diff
@@ -141,7 +143,7 @@ def write_bytes(data, urlpath, name_function=None, compression=None,
         raise NotImplementedError("Unknown protocol for writing %s (%s)" %
                                   (protocol, urlpath))
 
-    files = open_files_write(paths, **storage_options)
+    files = [(open_files_write, p, storage_options) for p in paths]
     out = [delayed(write_block_to_file)(v, f, compression, encoding)
            for (v, f) in zip(data, files)]
```

Comment:
Yeah but they're in the same task.  From the scheduler's perspective these two things are one unit.